src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Manifest.component.tsx

```diff
@@ -475,6 +540,136 @@ const ManifestComponent = ({
+                Icon={ICInfoFilled}
+                iconClass="icon-dim-16"
+                linkClass="dc__truncate--clamp-6"
+            />
+        )
+    }
+
+    const renderErrorBar = (isCodeEditorView: boolean = false) => {
+        if (showManifestCompareView || !errorText) {
+            return null
+        }
+
+        if (isCodeEditorView) {
+            return <CodeEditor.ErrorBar text={errorText} />
+        }
+
+        return (
+            <InfoColourBar
+                message={errorText}
+                classname="w-100 m-0 code-editor__error dc__no-border-radius dc__no-top-border dc__no-left-border dc__no-right-border dc__word-break"
+                Icon={ICErrorExclamation}
+                iconClass="icon-dim-16"
+                linkClass="dc__truncate--clamp-6"
+            />
+        )
+    }
+
+    const renderContent = () => {
+        if (!isReadOnlyView && manifestFormConfigurationType === ConfigurationType.GUI) {
+            return (
+                <>
+                    {renderEditorInfo()}
+                    {renderErrorBar()}
+                    <ManifestGUIView
+                        guiSchema={guiSchema}
+                        handleChange={handleGUIViewValueChange}
+                        // For uniformity have called method but as of now in this case it will always be trimedManifestEditorData
+                        manifestYAMLString={getCodeEditorValue()}
+                        handleSwitchToYAMLMode={handleSwitchToYAMLMode}
+                    />
+                </>
+            )
+        }
+
+        return (
+            <CodeEditor
+                defaultValue={showManifestCompareView && desiredManifest}
+                cleanData={showManifestCompareView}
+                diffView={showManifestCompareView}
+                theme="vs-dark--dt"
+                height={isResourceBrowserView ? 'calc(100vh - 119px)' : 'calc(100vh - 77px)'}
+                value={getCodeEditorValue()}
+                mode={MODES.YAML}
+                readOnly={isReadOnlyView}
+                onChange={handleEditorValueChange}
+                loading={loading}
+                customLoader={
+                    <MessageUI
+                        msg={loadingMsg}
+                        icon={MsgUIType.LOADING}
+                        size={24}
+                        minHeight={isResourceBrowserView ? 'calc(100vh - 151px)' : ''}
+                    />
+                }
+                focus={isEditMode}
+            >
+                {renderEditorInfo(true)}
+
+                {!loading &&
+                    !error &&
+                    isConfigDriftEnabled &&
+                    'hasDrift' in _selectedResource &&
+                    _selectedResource.hasDrift &&
+                    !showManifestCompareView &&
+                    renderOutOfSyncWarning &&
+                    renderOutOfSyncWarning(handleDesiredManifestOpen)}
```

Comment:
can add comment for this condition 
break down as well

Reply:
Not sure about this code snippet, since only migrated the code block may be @arunjaindev might be able to help